code/game/sound.dm

```diff
@@ -22,6 +26,65 @@
 		0, // Flags (1 = Auto Direct, 2 = Auto Room, 4 = Auto RoomHF)
 	)
 	environment = SOUND_ENVIRONMENT_NONE //Default to none so sounds without overrides dont get reverb
+	var/atom/attached
+	var/list/listeners = list()
+	var/maxdist = 0
+
+/sound/proc/attach( var/atom/towhat )
```

Comment:
that var would be better in snake case, remove var/ part as that is unnecessary

also remove the spaces to the left and right of it, lol